### DISCUSSION TITLE
Absence of localized data at the County level

### DISCUSSION THREAD

**Discussion by Domi2024** [lsc-hubs/kenya-hub#29] [General]
From the data catalogue there is hardly any data for Taita Taveta. This will limit the functionality of the hub in this region

**Comment by pvgenuchten** [comment]
Thank you for your comment @Domi2024, are you aware of any sources of data specific to Taita Taveta. Or do you expect this content is generated/collected as part of the project. 

Which aspects should have focus, when collecting data specifically for Taita Taveta (for example: certain crops, market value, climate, transportation options)

Or maybe you are referring to the use of local language, instead of english?
